modules/exploits/windows/iis/ms01_026_dbldecode.rb

```diff
@@ -49,8 +49,6 @@ def initialize(info = {})
         OptString.new('CMD', [ false, 'Execute this command instead of using command stager', nil ])
       ])
 
-    framework.events.add_exploit_subscriber(self)
```

Comment:
Note: This is not the fix required. It was temporarily removed for the purposes of ensuring that the `check` method of modules does not cause memory leaks anymore. I haven't investigated the specific implementation just yet, but I believe a similar approach to how payloads are handled might be required:

https://github.com/rapid7/metasploit-framework/blob/3b30b537725171d5fd8b9f441b9ab2440d84f3a8/lib/msf/core/payload.rb#L534-L538

Reply:
Going to go ahead with this fix, as it seems like the right solution for now.